tests/test_api_requestor.py

```diff
@@ -11,7 +11,7 @@
 from stripe._stripe_response import StripeResponse, StripeStreamResponse
 from stripe._encode import _api_encode
 
-from urllib.parse import urlsplit
+from urllib.parse import urlsplit, parse_qsl
```

Comment:
left over from before we made _util completely private. But there's no reason why we are re-exporting parse_qsl from util.